src/styles/blocks/card/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__box-1 {
+    margin-top: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    display: block;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__stats {
+    margin-top: 16px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &_reviews {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    line-height: 18px;
+    margin-top: 24px;
+
+    &-title {
+      display: block;
+      font-size: 12px;
+      color: $secondary-color;
+    }
+
+    &-value {
+      display: block;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: block;
+    margin-top: 16px;
+    color: $white-color;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 40px;
+    height: 40px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-align: center;
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+    transition: 0.4s;
```

Comment:
add color property

Reply:
color property was added in normalize.scss